bin/alluxio

```diff
@@ -11,7 +11,11 @@
 #
 
 function printUsage {
-  echo "Usage: alluxio COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo "Usage: alluxio [-jvmdebug] COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo
+  echo "Flag description:"
+  echo -e "  -jvmdebug     \t If this flag specified, the env variable ALLUXIO_USER_DEBUG_JAVA_OPTS will be added to jvm flags."
```

Comment:
@aaudiber I'm also agree to move this '-s' into the java implementation. And if sometime we want a shell level flag or argument like -debug, the -debug generic command flag will make trouble to parse other flag or argument, we have to judge if there are -debug before the generic command parse logic. If we shift it earlier, the parse logic will keep same to before.